### PR TITLE
refactor(ui): update language selectionscreen 

### PR DESCRIPTION
**Pull request by Arjun-sna**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | one plus5, Asus Zenfone |
| Bug fix?         | no      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  | #620        |

---

## Screenshots

| Before   | After    |
| -------- | -------- |
|before|after|

## Description
Added a new screen for updating user language settings

### CODE REVIEW

File: src/auth/screens/language-setting.screen.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          renderItem={({ item }) => {
+            return (
+              <StyledListItem
+                title={
+                  <View style={styles.language}>
+                    <Text style={styles.flag}>
+                      {emojifyText(item.emojiCode)}
+                    </Text>
+                    <Text style={styles.listTitle}>{item.name}</Text>
+                  </View>
```
Comment (ocarreterom) [inline src/auth/screens/language-setting.screen.js]: Use `styled`?

File: src/auth/screens/language-setting.screen.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  }
+
+  render() {
+    const { locale, changeLocale } = this.props;
+
+    return (
+      <ViewContainer>
+        <FlatList
+          data={languages}
+          renderItem={({ item }) => {
```
Comment (ocarreterom) [inline src/auth/screens/language-setting.screen.js]: Go to an independent function?

**Comment by machour** [comment]
@Arjun-sna I just merged #702 which migrated the user options screen to styled component.
I also took care of fixing the conflict for your PR.

Could you follow @ocarreterom sound advice and migrate `language-setting.screen.js` to be completely using styled components? 🙏

File: src/auth/screens/language-setting.screen.js (hunk 2 above)
Comment (Arjun-sna) [reply src/auth/screens/language-setting.screen.js]: Done.

File: src/auth/screens/language-setting.screen.js (hunk 1 above)
Comment (Arjun-sna) [reply src/auth/screens/language-setting.screen.js]: Done.

**Comment by machour** [comment]
Wonderfull work, thank you @Arjun-sna !

File: .all-contributorsrc [hunk 3]
```diff
@@ -693,6 +693,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "Arjun-sna",
+      "name": "Arjun",
```
Comment (housseindjirdeh) [inline .all-contributorsrc]: 🎉

**Comment by housseindjirdeh** [comment]
And thanks a million for reviewing @machour @ocarreterom 🙏

**Pull request merged**